documentation/book/con-what-the-topic-operator-does.adoc

```diff
@@ -0,0 +1,27 @@
+[id='what-the-topic-operator-does-{context}']
+= What the Topic Operator does
+
+The Topic Operator provides a way of managing topics in a Kafka cluster via {ProductPlatformName} resources.
```

Comment:
@ncbaratta, the value for attribute value for {ProductPlatformName} is {OpenShiftName} or {KubernetesName}. Do we use ifdef even here or is there any other best way to handle it?

Reply:
@dmuntima downstream the attribute will have the value {OpenShiftName}, so no need to resort to `ifdef` that I can see.